components/Common/Search/index.module.css

```diff
@@ -28,8 +28,10 @@
   @apply invisible
```

Comment:
we should make this element as flex an item-center

Reply:
Some stylings for this button bother me too, I'll take a look 👍